cpp/README.md

```diff
@@ -28,21 +28,36 @@ The main artifact produced by the build system is the shared library libcuml++.
 
 Current cmake offers the following configuration options:
 
+- Build Configuration Options:
+
 | Flag | Possible Values | Default Value | Behavior |
 | --- | --- | --- | --- |
-| BLAS_LIBRARIES | path/to/blas_lib | "" | Optional variable allowing to manually specify location of BLAS library. |
 | BUILD_CUML_CPP_LIBRARY | [ON, OFF]  | ON  | Enable/disable building libcuml++ shared library. Setting this variable to `OFF` sets the variables BUILD_CUML_TESTS, BUILD_CUML_MG_TESTS and BUILD_CUML_EXAMPLES to `OFF` |
 | BUILD_CUML_TESTS | [ON, OFF]  | ON  |  Enable/disable building cuML algorithm test executable `ml_test`.  |
-| BUILD_CUML_MG_TESTS | [ON, OFF]  | ON  |  Enable/disable building cuML algorithm test executable `ml_mg_test`. |
+| BUILD_CUML_MG_TESTS | [ON, OFF]  | ON  |  Enable/disable building cuML algorithm test executable `ml_mg_test`. Requires MPI installed and turning BUILD_CUML_MPI_COMMS to ON. |
```

Comment:
```suggestion
| BUILD_CUML_MG_TESTS | [ON, OFF]  | ON  |  Enable/disable building cuML algorithm test executable `ml_mg_test`. Requires MPI to be installed. When enabled, BUILD_CUML_MPI_COMMS will be automatically set to ON. |
```

It seems like this is might get confusing for folks. Since the BUILD_CUML_MPI_COMMS is always going to be ON when the tests are built, can we enable it autmoatically with this flag?

Reply:
@dantegd and I had a conversation on this one too, and we decided to go the other way round. Which is, both these flags need to be explicitly put to on. Building MPI comms by default when tests are built will create other issues such as needing an implementation of openmpi in the environment